usage/app/lib/Config.scala

```diff
@@ -31,6 +31,7 @@ object Config extends CommonPlayAppProperties with CommonPlayAppConfig {
   val defaultPageSize = 100
   val defaultMaxRetries = 6
   val defaultMaxPrintRequestSizeInKb = 500
+  val defaultDateLimit = "2016-01-01T00:00:00+00:00"
```

Comment:
Thought about it - i'd rather keep the defaults to primitives (as that's roughly what we'll be getting out of the properties files).